flytekit/types/structured/basic_dfs.py

```diff
@@ -76,7 +75,10 @@ def encode(
     ) -> literals.StructuredDataset:
         path = typing.cast(str, structured_dataset.uri) or ctx.file_access.get_random_remote_path()
         df = structured_dataset.dataframe
-        pq.write_table(df, path, filesystem=get_filesystem(path))
+        local_dir = ctx.file_access.get_random_local_directory()
+        local_path = os.path.join(local_dir, f"{0:05}")
```

Comment:
Can you leave a comment explaining why we need a file named `00000`? This is already present in `dataset_example.py` (I'm not sure why), but that doesn't have a comment either.

Reply:
will add a comment... this is only to keep the semantics the same as existing behavior.